frontend/routes/DummyRoute/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Route } from '@shopgate/pwa-common/components';
+import { Theme } from '@shopgate/pwa-common/context';
+import { DUMMY_PATH } from '../../constants';
+
+/**
+ * @returns {JSX}
+ */
+function DummyPage() {
+  return (
+    <Theme>
```

Comment:
i talked with szymon and we both think, we should use our extension kit here

Reply:
Updated to use extension kit

Reply:
Well, not really:
https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withThemeComponents

You're wrapping component with `withThemeComponents` but still you're using the Theme context.

Reply:
I see. Updated!